sphinx_doc/howtos/tutorials/docs/navigation2_on_real_turtlebot3.rst

```diff
@@ -0,0 +1,161 @@
+.. _navigation2-on-real-turtlebot3:
+
+Navigation2 on a Real Turtlebot 3
+*********************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+- `Videos`_
+
+Overview
+========
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot.
+Before completing this tutorials, completing the :ref:`navigation2-with-turtlebot3-in-gazebo`. is highly recommended especially if you are new to ROS and Navigation2.
+
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2).
+Turtblebot robots are widely supported by the ROS community.
+You can find many applications and examples of Turtlebot projects on the Internet.
+You can find more information about Turtlebot3 `here. <http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/>`_
+
+Links to the robot kits and Github repo.
+
+- `Turtlebot Burger <http://www.robotis.us/turtlebot-3-burger-us/>`_
+- `Turtlebot Waffle <http://www.robotis.us/turtlebot-3-waffle-pi/>`_
+- `Turtlebot 3 Github <https://github.com/ROBOTIS-GIT/turtlebot3>`_
+
+This tutorial consists of two parts. In the first part, you learned how to use Turtlebot 3 robots in simulation (Gazebo).
+In this part, you will learn how to control a real Turtlebot Waffle using Navigation2.
+
+``ROS2 Dashing`` and ``Navigation2 Dashing 0.2.4`` are used to create this tutorial.
+You should be able to do this tutorial using other Navigation2 versions as well.
+
+This tutorial may take about 1-2 hours to complete. 
+It depends on your experience with ROS, robots, and what computer system you have.
+
+Requirements
```

Comment:
If they've gone through the gazebo tutorial, this should be all done.

I'd recommend saying something like "you must install Navigation2, turtelbot3. If you don't have them installed from the previous tutorial, reference it for this step" for 2 reasons

- Maintenance, the less redundancy we have, the easier to keep docs in sync. With that, I'd also be OK if there was a way to link the Requirements section of the gazebo tutorial to this one so that there's only 1 time listed.
- Repetition. I think the tutorials should be viewed as ordered things to go through so you can assume knowledge of the prior information in the later information.